stdlib/public/core/Builtin.swift

```diff
@@ -983,8 +983,8 @@ public func type<T: ~Copyable & ~Escapable, Metatype>(
   Builtin.unreachable()
 }
 
+@abi(func type<T, Metatype>(of _: T) -> Metatype)
 @_spi(SwiftStdlibLegacyABI) @available(swift, obsoleted: 1)
-@_silgen_name("$ss4type2ofq_x_tr0_lF")
```

Comment:
I think I would prefer to just remove these symbols (in Builtin.swift) altogether since no one should be linking them in the first place.